pkg/provider/gcp.go

```diff
@@ -270,6 +271,14 @@ func (gcp *GCPProvider) mkBucket(name string) error {
 	return nil
 }
 
+func (gcp *GCPProvider) clusterLocation() (string, string) {
+	if z, ok := gcp.ctx["clusterZone"]; ok {
```

Comment:
Shouldn't we be setting this in the context somewhere?

Reply:
it has to be done manually, and we'll document it as part of the process

Reply:
We should be able to handle that more gracefully but it’s fine for now.